### ISSUE TITLE
Update storybook peer dependency to v2

### ISSUE THREAD

**Issue by rcoedo** [Tarabyte/react-storybook-addon-intl#2]
The storybook dependency is outdated even though the add-on seems to work with the latest version. I get an ugly `UNMET PEER DEPENDENCY @kadira/storybook@2.26.0` when I try to use it.

Do you plan on updating this package? It seems quite good!

Thanks! 👍

**Comment by Tarabyte** [comment]
> Do you plan on updating this package?

Yep. I'm planning to update to the latest version of storybook later this week.

**Comment by lauterry** [comment]
Hello

Any update on this issue ?

**Comment by dmytro-lymarenko** [comment]
Hi. How is it going with the updating? :)
